grin/grin.cabal

```diff
@@ -130,8 +130,9 @@ library
     monad-gen,
     free,
     bytestring,
-    llvm-hs,
-    llvm-hs-pure,
+    llvm-hs ==5.*,
```

Comment:
GRIN uses llvm 7
https://www.stackage.org/nightly-2018-11-07/package/llvm-hs-7.0.1
https://github.com/grin-tech/grin/blob/master/stack.yaml#L1

Reply:
Do not specify the version in the cabal file.
Stack fixes the versions.

Reply:
Okay thanks, I'll fix things up accordingly.

Reply:
@csabahruska I don't use stack to build my code. Do you really prefer I leave the versions out of the cabal file?

Reply:
Can't you set the version in the nix file?

Reply:
Nix works without putting the version in .cabal. It's just by having the version in the cabal file, cabal will tell me if I nix uses the wrong version by mistake.

Reply:
Also, I'd suggest putting a lower bound on `base` because this codebase doesn't build on GHC8.2